frontend/package.json

```diff
@@ -10,12 +10,15 @@
   },
   "scripts": {
     "build": "nuxt build",
+    "predev": "npm run generateDocs",
     "dev": "nuxt dev",
     "generate": "nuxt generate",
     "preview": "nuxt preview",
-    "postinstall": "nuxt prepare",
+    "postinstall": "nuxt prepare && npm run generateDocs",
+    "postupdate": "npm run generateDocs",
     "lint": "eslint --max-warnings 0 .",
     "typecheck": "nuxt typecheck",
+    "prestart": "npm run generateDocs",
```

Comment:
Based on the situation described above and to simplify dev experience (not requiring them to `cd api && npm i && cd ../frontend`), I would suggest to:
- `A` only generate docs inside docker (so probably only keep `prestart` here and remove hard requirement for the content/md file to run `npm run dev`)
- `B` use something like https://www.npmjs.com/package/live-plugin-manager to install / download package from the `api/package.json` locally – this is more like a crazy hack, so I wouldn't recommend it

Reply:
1. Frontend docker file already copies the api folder 
  - i've added the `file:` dependency in the `package.json` of frontend.
2. The Frontend dockerfile does not use `npm start` instead it goes with `
CMD [ "node", "/app/frontend/.output/server/index.mjs" ]
`
    - therefore i went with `postbuild`

Reply:
Dockerfile for frontend does not use `npm start` so i've added the documentation generation on `prebuild`